### PR TITLE
Update page

### PR DESCRIPTION
**Pull request by michaelbausor**
This is split by commit:
- Core
- BigQuery, Compute, Dns
- Reporting, monitoring, logging
- Pubsub, resourcemanager, spanner, storage, trace
- Examples

### CODE REVIEW

File: google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryResult.java [hunk 1]
```diff
@@ -144,12 +143,12 @@ public QueryResult getNextPage() {
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
-        .add("rows", getValues())
+        .add("rows", this)
```
Comment (garrettjonesgoogle) [inline google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryResult.java]: This doesn't seem right.

File: google-cloud-core/src/main/java/com/google/cloud/AsyncPage.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 package com.google.cloud;
 
 import com.google.api.gax.core.ApiFuture;
+import com.google.api.gax.core.Page;
```
Comment (garrettjonesgoogle) [inline google-cloud-core/src/main/java/com/google/cloud/AsyncPage.java]: Shouldn't the `AsyncPage` here be pulled out and replaced with the new `AsyncPage` from gax?

File: google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/BigQueryExample.java [hunk 3]
```diff
@@ -173,9 +173,8 @@ Void parse(String... args) throws Exception {
   private static class ListDatasetsAction extends NoArgsAction {
     @Override
     public void run(BigQuery bigquery, Void arg) {
-      Iterator<Dataset> datasetInfoIterator = bigquery.listDatasets().iterateAll();
-      while (datasetInfoIterator.hasNext()) {
-        System.out.println(datasetInfoIterator.next());
+      for(Dataset dataset : bigquery.listDatasets().iterateAll()) {
```
Comment (garrettjonesgoogle) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/BigQueryExample.java]: add space after `for` (multiple places)

File: google-cloud-examples/src/main/java/com/google/cloud/examples/logging/LoggingExample.java [hunk 4]
```diff
@@ -40,6 +41,7 @@
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
+import sun.rmi.runtime.Log;
```
Comment (garrettjonesgoogle) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/logging/LoggingExample.java]: Remove import

File: google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryResult.java (hunk 1 above)
Comment (michaelbausor) [inline google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryResult.java]: This is an incorrect change, fixed.

File: google-cloud-core/src/main/java/com/google/cloud/AsyncPage.java (hunk 2 above)
Comment (michaelbausor) [inline google-cloud-core/src/main/java/com/google/cloud/AsyncPage.java]: Yes, that is now done

File: google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/BigQueryExample.java (hunk 3 above)
Comment (michaelbausor) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/BigQueryExample.java]: Done

File: google-cloud-examples/.classpath [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (garrettjonesgoogle) [inline google-cloud-examples/.classpath]: Take this file out

File: google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java [hunk 6]
```diff
@@ -22,10 +22,10 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
-import com.google.cloud.AsyncPage;
+import com.google.api.gax.core.AsyncPage;
 import com.google.cloud.MonitoredResource;
 import com.google.cloud.MonitoredResourceDescriptor;
-import com.google.cloud.Page;
+import com.google.api.gax.core.Page;
```
Comment (garrettjonesgoogle) [inline google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java]: These imports need to be reordered

**Pull request merged**